.changeset/short-bikes-nail.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
change sets are required to produce new versions of the extension. Please remove the change set, as changes to readme/documentation should not result in new extension versions.